projects/aca-content/src/lib/components/search/search-navigation.service.spec.ts

```diff
@@ -62,4 +62,12 @@ describe('SearchNavigationService', () => {
 
     expect(routerNavigate).toHaveBeenCalledWith(['/search']);
   });
+
+  it('should navigate back to the previous route when the "navigationBack" button is clicked', () => {
```

Comment:
I think that title should be a bit different. In your test you don't click any button, you just call navigateBack function

Reply:
still not addressed